### DISCUSSION TITLE
Add option to use a custom character for progress bar

### DISCUSSION THREAD

**Discussion by xmstspider** [curl/curl#15458] [Ideas]
Hello,

Curl progress bar (-#) uses a hard-coded '#' character. 

Would it be possible to add a new switch to support an user-customizable character, maybe even a UTF-8 one?

Thank you

**Comment by bagder** [comment]
> Would it be possible to add a new switch to support an user-customizable character, maybe even a UTF-8 one?

Since it's just source code, virtually everything is possible, The question is possibly how it should be set by a user. And if it really is worth adding yet another command line option for this.

**Reply by xmstspider** [reply]
Thank you. I wondered, if there is the particular switch (-#) already, so possibly just to extend this one as a key-value option, e.g. ``-# '*'``, or ``--progress-bar='\u23f5'``, or something like that.

**Comment by abelcha** [comment]
Such an underrated feature, some slight customisation would make the experience much more pleasant
